src/app/components/media/MediaLog.js

```diff
@@ -145,6 +147,9 @@ const MediaLogContainer = Relay.createContainer(withPusher(MediaLogComponent), {
                 dbid,
                 name,
                 is_active,
+                team_user(team_slug: $teamSlug) {
+                  ${ProfileLink.getFragment('teamUser')},
```

Comment:
Could you please add a comment saying why this is here?

I can see by looking at the rest of the pull request that it's because the dream is to include `fragment on ProjectMedia { ${Annotations.getFragment('projectMedia')} }` but we aren't there yet. Without the context of this pull request, I'd never know.